image_writers_guide/metadata.adoc

```diff
@@ -9,96 +9,63 @@
-Docker image wants 'mysql' and 'redis' and you don't have the Docker image with
-'redis' tag, then UI might suggest you to add this image into your application.
-
-The format of the values is the same as for `IMAGE_TAGS`.
+|[envvar]#IMAGE_WANTS#
+|Specifies a list of tags that the UI and generation tools might suggest to provide if you do not have the Docker images with given tags already. For example, if the image wants `mysql` and `redis`, and you do not have an image with a `redis` tag, then the UI might suggest you to add this image into your application. Example format:
 
-=== IMAGE_DESCRIPTION
-
-This variable is used to give the image consumers more detailed description
-about the service or functionality given image provides. The UI can use this
-description together with the image name to give users more informations about
-the image.
+----
+IMAGE_WANTS=mysql,redis
+----
 
-Example:
+|[envvar]#IMAGE_DESCRIPTION#
+|Gives image consumers a more detailed description about the service or functionality provided by the image. The UI can use this description together with the image name to give users more information about the image. Example format:
 
 ----
 IMAGE_DESCRIPTION="MySQL 5.5 database"
 ----
 
-=== IMAGE_EXPOSE_SERVICES
+|[envvar]#IMAGE_EXPOSE_SERVICES#
+|Contains a list of ports that match with the `EXPOSE` instructions in the [filename]#Dockerfile# and provides more descriptive information about what actual service the given port provides.
 
-This variable contains a list of ports that match with the EXPOSE instructions
-in Dockerfile and provide more descriptive information about what actual service
-the given port provides.
-
-The format is `PORT[/PROTO]:SERVICE_NAME` where the `[PROTO]` part is optional
-and it defaults to `tcp` if it is not specified.
-
-Example:
+The format is `PORT[/PROTO]:SERVICE_NAME` where `[PROTO]` is optional and defaults to `tcp` if unspecified. Example format:
 
 ----
 IMAGE_EXPOSE_SERVICES="3128/tcp:mysql,8080:http"
 ----
 
-=== IMAGE_NON_SCALABLE
-
-An image might use this variable to suggest that it does not support scaling.
-The UI will then communicate this to consumers of that image.
-Being not-scalable basically means that the value of 'replicas' should initially
-not be set higher than 1.
-
-=== IMAGE_MIN_CPU and IMAGE_MIN_MEMORY
-
-This variable suggests how much resources the Docker image might need in order
-to work properly. The UI might warn the user that deploying this Docker image
-may exceed the user quota.
-
-The values must be compatible with [Kubernetes
-quantity](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/resources.md#resource-quantities) values for CPU and memory.
+|[envvar]#IMAGE_NON_SCALABLE#
+|Specifies that the image does not support scaling. The UI then communicates this to consumers of that image. Being non-scalable basically means that the value of `replicas` should initially not be set higher than `1`.
```

Comment:
@mfojtik what value do we want this to be set to?  presumably true/false.  we should indicate that.